draft-ietf-quic-multipath.md

```diff
@@ -127,11 +127,36 @@ path management in {{Section 9 of QUIC-TRANSPORT}} and therefore
 requires negotiation between the two endpoints using a new transport
 parameter, as specified in {{nego}}.
 
-This extension uses multiple packet number spaces.
-When multipath is negotiated, each separate packet number space is linked to a path ID.
+This extension specifies a new Path Identifier (Path ID), which is an
+integer between 0 and 2^32 - 1 (inclusive). Path identifies are generated
+monotonically increasing and cannot be reused.
+
+The Path ID is used to
+address a path in the new multipath control frames,
+such as PATH_ABANDON {{path-abandon-frame}}, PATH_STANDBY {{path-standby-frame}}},
+PATH_AVAILABLE {{path-available-frame}} as well as ACK_MP {{ack-mp-frame}}.
+Further, Connection IDs are issued per Path ID.
+Each connection ID is associated with one path identifier
+but multiple connection IDs can be associated with the same path identifier.
+
+The Path ID of the initial path is 0. Connection IDs
+which are issued by a NEW_CONNECTION_ID frame {{Section 19.15. of QUIC-TRANSPORT}}
+respectively are asociated with Path ID 0. Also, the Path ID for
+the connection ID specified in the "preferred address" transport parameter is 0.
```

Comment:
```suggestion
the connection ID specified in the "preferred address" transport parameter is 2.
```

Reply:
I think this change needs discussion. See issue #317